library/Ivoz/Provider/Domain/Model/Fax/FaxDto.php

```diff
@@ -41,4 +64,39 @@ public function denormalize(array $data, string $context, string $role = ''): vo
             $data
         );
     }
+
+    /**
+     * @return array<array-key, mixed>
+     */
+    public function normalize(string $context, string $role = ''): array
+    {
+
+        $response = parent::normalize($context, $role);
+
+        if (in_array($context, self::CONTEXTS_WITH_REL_USERS, true)) {
+            $response['relUserIds'] = $this->relUserIds;
+        }
+
+        return $response;
+    }
+
+    /**
+     * @param int[] $userIds
+     *
+     * @return void
+     */
+    public function setRelUserIds(array $userIds): void
+    {
+        $this->relUserIds = $userIds;
+
+        $faxesRelUsers = [];
+        foreach ($userIds as $id) {
+            $dto = new FaxesRelUserDto();
+            $dto->setUserId($id);
```

Comment:
```suggestion
            $dto = new FaxesRelUserDto($id);
```
just if you prefer, no need to change it

Reply:
Not the same, but I notice $id should be $userId.